fastcrud/endpoint/endpoint_creator.py

```diff
@@ -249,9 +269,13 @@ def _read_item(self):
 
         @apply_model_pk(**self._primary_keys_types)
         async def endpoint(db: AsyncSession = Depends(self.session), **pkeys):
-            item = await self.crud.get(db, **pkeys)
+            if stmt := self._base_read_statement():
```

Comment:
This way, while we're trying to make FastCRUD as useful as possible, we're not forcing anyone to use either FastCRUD or EndpointCreator. The latter may be used with raw SQLAlchemy querries and the former can be used in your usual single FastAPI endpoints. And of course you will probably get most by combining the 2 features, but you're never going into a situation where you're limited by the library with something you could achieve with raw SQLAlchemy.

We could do the same with other methods, especially to easily implement complex joins and filters.